week-1-ts-ecosystem/w01e01-typescript-basics/src/model.ts

```diff
@@ -0,0 +1,13 @@
+import {User} from "./types";
+
+const user: User = {
+  name: 'Andy',
+  age: 30,
+  email: 'andy@mail-me-tommorow.com',
+  address: {
+    street: 'Strange Alley',
+    no: 23,
+  },
+}
+
+export {user}
```

Comment:
W mojej opinii to nie jest `model` tylko `data`

no i nie koniecznie bym to nazwał `data.ts` - to "zbyt ogólne"
może być:
`user.data.ts` jeśli już np.

U ciebie `model.ts` to to co masz w `types.ts`
Model to kształt danych (co nie zawsze musi być klasą - u Ciebie są tam (w `types.ts`) interface - i to jest ok)